native/src/routes/settings/components/Settings.js

```diff
@@ -1,16 +1,15 @@
 // @flow
```

Comment:
Refactoring only, moved some functionality to `SettingsItems` here: badges and switches